### ISSUE TITLE
Add another Window to the "Different Windows" Item

### ISSUE THREAD

**Issue by hop193** [BEEmod/BEE2-items#1820]
By "Different Windows" I mean these:
![kys](https://cloud.githubusercontent.com/assets/25728252/24819053/02d7c082-1bb1-11e7-9c1f-0a5f46440360.PNG)
And to add one where half of the wall is (non)portable and the other half is grating/window. Use something like 'Start Enabled' or something to reverse the orientation.

I guess you could replace the last one. (If you had to replace one that is)

**Comment by Luke18033** [comment]
Unfortunately, Button Type only supports 3 options.

**Comment by Luke18033** [comment]
Actually, I think we could use Cube Type. That would allow us to have 5 variants.

**Comment by hop193** [comment]
> Actually, I think we could use Cube Type. That would allow us to have 5 variants.

Yes! That would work. But what could be the fifth? Hmm.
Come to think of it, we could use angles to set the windows then. (30, 45, 60, 90)

**Comment by Endermage77** [comment]
Perhaps the fifth variant could be a horizontal variant of the first window there?

**Comment by awesomedoctorx3** [comment]
maybe the fifth is just a piece of glass, no sides? this would be different from normal glass, because these can be positioned in more positions?

**Comment by Endermage77** [comment]
If we wanted exactly 4 variants, we could do Gel type too.

**Comment by Luke18033** [comment]
Nope, we can't use either of those. Cube Type is the only "type" property supported by all item classes.

**Comment by hop193** [comment]
> Nope, we can't use either of those. Cube Type is the only "type" property supported by all item classes.

Really? Like the 5-positioning system lasers use? The more you know.

**Comment by hop193** [comment]
But the fifth one could be to move the window (like the second one) to the top. I wouldn't really enjoy it, (heck I don't even use the third one) but I mean I'm not the one who decides which items get in and don't. That's more up to people like @Luke18033 , @LautaroL20, and mainly @TeamSpen210.

**Comment by TeamSpen210** [comment]
The fifth would be your half design reversed I think.

**Comment by hop193** [comment]
Oh, that would actually work out.
